### PR TITLE
[docs] indent

### PR DESCRIPTION
**Pull request by missinglink**
minor minor indentation issue

### CODE REVIEW

File: docs/reference/index-modules/similarity.asciidoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ PUT /index
                 "normalization" : "h2",
                 "normalization.h2.c" : "3.0"
               }
-            }
+           }
```
Comment (jrodewig) [inline docs/reference/index-modules/similarity.asciidoc]: Github won't allow me to comment on the line, but do you mind fixing the indentation on line 27 (` "similarity"`) as well?

It looks like it uses tab/5 spaces.

**Review by jrodewig — CHANGES_REQUESTED** [review]
Left a request to fix one other indentation issue. After that's addressed, I can merge and backport.

**Comment by missinglink** [comment]
@jrodewig this is not a new code submission and I don't work for elastic.
I'm happy to continue catching and fixing docs issues for you but I'm not working for you for free.

If you see another issue in the same document you can open a PR yourself to fix it rather than expecting me to do it for you.
If you need help using the `cherry-pick` command I am happy to oblige.

**Comment by jrodewig** [comment]
Hi @missinglink.

Sorry for any confusion. The change I requested is indeed related to this one.

Line 27 contains the opening bracket (`{`) for the change you submitted. Unfortunately, I cannot accept your changes without also fixing the indentation for that line.

I'm going to close this PR and treat this as a bug report. Please feel free to re-open this PR if you'd like to address.

Thanks for raising this issue.

**Pull request closed without merging**